Components/Card/Card.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="Card.css">
+  <title>Perry's Card</title>
+</head>
+
+<body>
+
+<div id="card">
+  <img src="images/20240413_095423.jpg" alt="An image of perry chamileke smiling in a cap">
+  <article>
+    <h4>Perry Chamileke</h4>
```

Comment:
Headlines always need to stay in hierarchical order. That's why I would rather use an H2 here. This is with the Idea in mind, that later on the full website, at least one H1 will be present.